web/src/bootstrap_typeahead.ts

```diff
@@ -658,6 +661,11 @@ export class Typeahead<ItemType extends string | object> {
                 ) {
                     return;
                 }
+                if (pseudo_keycode === 67 && e.ctrlKey && e.shiftKey) {
```

Comment:
There would be very few shortcuts which would interfere with the typeahead, so I found it okay to put it here.

Reply:
Yep, seems correct to put it here.